### PR TITLE
Jk local storage

### PR DESCRIPTION
**Pull request by jenkens-dev**

### CODE REVIEW

**Comment by jenkens-dev** [comment]
@alphy11

File: src/components/DropdownOptions.tsx [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
           className="fill-current text-blue-500 h-6 w-6"
           xmlns="http://www.w3.org/2000/svg"
-          viewBox="0 0 20 20"
-        >
+          viewBox="0 0 20 20">
           <path d="M4.13 12H4a2 2 0 1 0 1.8 1.11L7.86 10a2.03 2.03 0 0 0 .65-.07l1.55 1.55a2 2 0 1 0 3.72-.37L15.87 8H16a2 2 0 1 0-1.8-1.11L12.14 10a2.03 2.03 0 0 0-.65.07L9.93 8.52a2 2 0 1 0-3.72.37L4.13 12zM0 4c0-1.1.9-2 2-2h16a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V4z" />
-        </svg>
-      </DropdownLink>
-      <DropdownLink hrefLink="/" linkText="Logout">
+        </svg>{' '}
```
Comment (Alphy11) [inline src/components/DropdownOptions.tsx:29]: extranneous space

File: src/components/DropdownOptions.tsx [hunk 2]
```diff
@@ -1,37 +1,45 @@
...
         <svg
           className="fill-current text-blue-500 h-6 w-6"
           xmlns="http://www.w3.org/2000/svg"
-          viewBox="0 0 20 20"
-        >
+          viewBox="0 0 20 20">
           <path d="M10 0a10 10 0 1 1 0 20 10 10 0 0 1 0-20zM2 10a8 8 0 1 0 16 0 8 8 0 0 0-16 0zm10.54.7L9 14.25l-1.41-1.41L10.4 10 7.6 7.17 9 5.76 13.24 10l-.7.7z" />
-        </svg>
-      </DropdownLink>
+        </svg>{' '}
```
Comment (Alphy11) [inline src/components/DropdownOptions.tsx:40]: extranneous space

File: src/components/Navbar.tsx [hunk 3]
```diff
@@ -8,6 +8,17 @@ const Navbar = () => {
   const [expanded, setExpanded] = useState(false);
   const [loggedIn, setLoggedIn] = useState(false);
 
+  useEffect(() => {
+    if (
+      localStorage.getItem('volunteerId') ||
```
Comment (Alphy11) [inline src/components/Navbar.tsx:13]: typically you'll use cookies here but 🤷 it's probably fine

File: src/pages/VolunteerDashboardPage.tsx [hunk 4]
```diff
@@ -15,20 +15,18 @@ const GET_VOLUNTEER = gql`
...
+  const volunteerId = parseInt(localStorage.getItem('volunteerId')!);
+
+  const { error, data } = useQuery<getVolunteer>(GET_VOLUNTEER, {
     variables: {
-      id: volunteer.volunteerID,
+      id: volunteerId,
     },
   });
+
+  if (error) {
```
Comment (Alphy11) [inline src/pages/VolunteerDashboardPage.tsx:26]: forgot to delete

**Review by Alphy11 — COMMENTED** [review]
Sorry about the delay.  Nothing significant to call out here.

**Pull request merged**